tests/array.js

```diff
@@ -2,26 +2,24 @@
 /* jshint esversion: 6 */
 /* jshint -W030 */
 
-var chai = require('chai');
-var expect = chai.expect;
-var jsonexport = require('../lib/index');
-var os = require('os');
```

Comment:
Updated the existing test to check for 0. But noticed that, when it failed, it did not report a test failure. Instead there was an `UnhandledPromiseRejectionWarning` and all tests _passed_. i.e. As the callback was invoked after the test had completed.

Refactoring to report failures.

Reply:
Yeah, i noticed that as well, thanks for fixing it!